src/Domain/Creators/TimeDependentOptions/ExpansionMap.hpp

```diff
@@ -0,0 +1,98 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <optional>
+#include <string>
+#include <variant>
+
+#include "DataStructures/DataVector.hpp"
+#include "Domain/Creators/TimeDependentOptions/FromVolumeFile.hpp"
+#include "Options/Auto.hpp"
+#include "Options/Context.hpp"
+#include "Options/String.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace domain::creators::time_dependent_options {
+/*!
+ * \brief Class to be used as an option for initializing expansion map
+ * coefficients.
+ */
+struct ExpansionMapOptions {
+  using type = Options::Auto<ExpansionMapOptions, Options::AutoLabel::None>;
+  static std::string name() { return "ExpansionMap"; }
+  static constexpr Options::String help = {
+      "Options for a time-dependent expansion of the coordinates. Specify "
+      "'None' to not use this map."};
+
+  struct InitialValues {
+    using type =
+        std::variant<std::array<double, 3>, FromVolumeFile<names::Expansion>>;
+    static constexpr Options::String help = {
+        "Initial values for the expansion map, its velocity and "
+        "acceleration."};
+  };
+
+  struct InitialValuesOuterBoundary {
+    using type =
+        std::variant<std::array<double, 3>, FromVolumeFile<names::Expansion>>;
+    static constexpr Options::String help = {
+        "Initial values for the expansion map, its velocity and "
+        "acceleration at the outer boundary. Unless you are starting from a "
+        "checkpoint or continuing an evolution, this option should likely be "
+        "[1.0, 0.0, 0.0] at the start of an evolution"};
+  };
+
+  struct DecayTimescaleOuterBoundary {
+    using type = double;
```

Comment:
Are these options not read from the vol file?

Reply:
Currently no. But I could edit the FoT to expose these values so we can copy them in. That's probably the correct thing to do here.